d3rlpy/algos/base.py

```diff
@@ -141,6 +141,7 @@ def fit_online(self,
                    update_start_step=0,
                    eval_env=None,
                    eval_epsilon=0.0,
+                   save_metrics=True,
```

Comment:
is this related to SB3?

Reply:
Actually, this is not related to SB3 wrapper. But, let me include this in this PR.

Reply:
ok, no pb ;)